llvm/include/llvm/TargetParser/RISCVTargetParser.h

```diff
@@ -32,6 +32,21 @@ struct RISCVExtensionBitmask {
 };
 } // namespace RISCVExtensionBitmaskTable
 
+struct CPUModel {
+  uint32_t MVendorID;
+  uint64_t MArchID;
+  uint64_t MImpID;
```

Comment:
MImplID to match the name of the CSR?

Reply:
Nevermind, the CSR is mimpid. hwprobe is the one that added the 'l' to the name.